db/seeds.d/106-mail_notifications.rb

```diff
@@ -5,6 +5,12 @@
 #
 ::User.current = ::User.anonymous_api_admin
 
+# The notification names are used as humanized labels. These need to be
+# translated as well as the description
+N_('Host errata advisory')
+N_('Sync errata')
+N_('Promote errata')
+
```

Comment:
would it make sense to just do:
```
notifications.each{ |notif| N_(notif[:name].to_s.gsub('_', ' ').capitalize ) }
```

or would that not get picked up by the translations?